openapi.yaml

```diff
@@ -331,6 +331,59 @@ paths:
           description: CashLetter deleted
         '404':
           description: CashLetter or File not found
+  /v2/files:
+    post:
+      tags: ['Image Cash Letter Files']
+      summary: Create file
+      operationId: createICLFileV2
+      requestBody:
```

Comment:
Does OpenAPI have a `paramteres:` object which can specify headers we look at? We should clarify that `Accept:` is also used by this endpoint.

Reply:
It does have `parameters`, but the docs say you're not allowed to include `Accept` and a couple other headers in them. Instead, I added "Use the `Accept` header to specify the response format." to the request description.
![image](https://github.com/moov-io/imagecashletter/assets/11085638/c3bc48a2-85d7-4ae2-aafe-7a091e172d5a)

Reply:
Ah okay. I wonder if it's because the OpenAPI generated client checks Accept and sets up JSON, XML, etc decoding automatically.